docs(data exchange): adapt and restructure the data exchange (#177)
eclipse-tractusx/tractus-x-umbrella: docs/user/guides/data-exchange/provide-data.md

  }
}
```

### Step 2: Validate the asset

Check that the asset was created successfully by running the following for requesting the specific asset with ID 200:

```bash
curl -L -X GET 'http://dataprovider-controlplane.tx.test/management/v3/assets/200' \
-H 'X-Api-Key: TEST2' | jq
```

Alternatively, retrieve all assets:

```bash
curl -L -X POST 'http://dataprovider-controlplane.tx.test/management/v3/assets/request' \
-H "x-api-key: TEST2" \
-H "content-type: application/json" | jq
```

#### Expected output

The result for the first command shows just the newly created asset:

```json
{
  "@id": "200",
  "@type": "Asset",
  "properties": {
    "description": "Product EDC Demo Asset",
    "id": "200"
  },
  "dataAddress": {
    "@type": "DataAddress",
    "type": "HttpData",
    "baseUrl": "https://jsonplaceholder.typicode.com/todos/3"
  },
  "@context": {
    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
    "edc": "https://w3id.org/edc/v0.0.1/ns/",
    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
    "tx-auth": "https://w3id.org/tractusx/auth/",
    "cx-policy": "https://w3id.org/catenax/policy/",
    "odrl": "http://www.w3.org/ns/odrl/2/"
  }
}
```

> The second request will return all assets. It`s called `request catalog` and will be explained in the next step.

Reviewer: ```suggestion
The second request will return all assets. It's called `request catalog` and will be explained in the next step.
```

## Request the catalog

> Once the asset is created, Alice (the data consumer) can try request a catalog of available (offered) data.

Reviewer: ```suggestion
Once the asset is created, Alice (the data consumer) can try to request a catalog of available (offered) data.
```

          "odrl:constraint": {
            "@type": "LogicalConstraint",
            "odrl:or": [
              {
                "@type": "Constraint",
                "odrl:leftOperand": {
                  "@id": "BusinessPartnerNumber"
                },
                "odrl:operator": {
                  "@id": "odrl:eq"
                },
                "odrl:rightOperand": "BPNL00000003AZQP"
              }
            ]
          }
        }
      ]
    }
  }' | jq
```

In this case a **Access Policy** is created. The assigned asset will only be visible as a data offer for the Business Partner Number `BPNL00000003AZQP`.

#### Expected Output

The policy was successfully created, if the response is something like this

```json
{
  "@type": "IdResponse",
  "@id": "200",
  "createdAt": 1732640748595,
  "@context": {
    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
    "edc": "https://w3id.org/edc/v0.0.1/ns/",
    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
    "tx-auth": "https://w3id.org/tractusx/auth/",
    "cx-policy": "https://w3id.org/catenax/policy/",
    "odrl": "http://www.w3.org/ns/odrl/2/"
  }
}
```

### Step 2: Request catalog - second try

> Now that Bob created an access policy, Alice can once again try to access Bob's offer by running this `curl`.

Reviewer: ```suggestion
Now that Bob has created an access policy, Alice can once again try to access Bob's offer by running this `curl`.
```
